tests/umockcallrecorder_ut/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ umockcallrecorder_ut.c
 
 set(${theseTestsName}_c_files
 umockcallrecorder_mocked.c
+../../src/
```

Comment:
does this add all of the source files in that path? Or just a mistake (line should be removed)?

Reply:
Removed, was a mistake

---
In reply to: [530656573](https://github.com/Azure/umock-c/pull/177#discussion_r530656573) [](ancestors = 530656573)